src/components/input-switch/foreignkey-field.tsx

```diff
@@ -147,7 +149,18 @@ const ForeignkeyField = (props: ForeignkeyFieldProps): JSX.Element => {
       getValues
     );
 
+    let currentSelectedRow;
```

Comment:
One issue with doing this is in the case of edit when a value is set for the input without making a selection in the modal. 

We need the `uniqueId` of the "selected row" to ensure it is the row to be selected in the modal. I need to think more about this. In this case, the `uniqueId` needs to be stored somewhere alongside the display value that is set in react hook form, maybe another value in RHF `values` specifically for the `uniqueId`.

If the `uniqueId` is tracked differently, then the implementation should change where bulk foreign key selections should set the display and uniqueId so the row is "set up" already instead of being fetched from `bulkForeignKeySelectedRows`.